Yoast/wordpress-seo
# Improves translatability of social appearance upsell label

## Context


*  The previous string was problematic because the words "Social" and "Twitter" can be translated differently in some languages depending on how they're used in a sentence.

## Summary


This PR can be summarized in the following changelog entry:

* Improves the translatability of social appearance upsell label.

## Relevant technical choices:

* Change the %1$s share preview string in the SocialUpsell.js file so that it doesn't use a variable. Instead we should have two strings: Social share preview and Twitter share preview.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

For devs: before testing in the plugin, run yarn test and make sure all tests pass.

Testers:
* Build Yoast SEO Free
* Open a post or page
* Confirm that in the metabox (under Social tab) "Social share preview" and "Twitter share preview" look as they previously did

#### Relevant test scenarios


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes #20723
